bin/guard

```diff
@@ -41,10 +41,6 @@ class GuardReloader
       Gem.win_platform?
     end
 
-    def default_system_ruby
-      RbConfig.ruby
-    end
```

Comment:
I think we should keep this method but probably rename it to `ruby` and handle Windows in it:

```ruby
def ruby
  config.windows? ? RbConfig.ruby : Gem.ruby
end
```

Reply:
I don't have a Windows box at hand, but if its RbConfig is anything like the one on MacOS, then it returns the default Ruby, not the one it's actually running.

Reply:
Then again, I seem to have mischecked. While they both seem to return the running Ruby, they return different paths. When installed through Homebrew, for example:
```
% irb24
>> RbConfig.ruby
=> "/usr/local/Cellar/ruby/2.4.2/bin/ruby24"
>> Gem.ruby
=> "/usr/local/opt/ruby/bin/ruby24"
```

Reply:
> if its RbConfig is anything like the one on MacOS, then it returns the default Ruby, not the one it's actually running.

FWIW I get:

```
› pry
[1] pry(main)> RbConfig.ruby
=> "/Users/remy/.rubies/ruby-2.3.3/bin/ruby"
[2] pry(main)> Gem.ruby
=> "/Users/remy/.rubies/ruby-2.3.3/bin/ruby"
```

And then:

```
› chruby 2.3.4
› pry
[1] pry(main)> RbConfig.ruby
=> "/Users/remy/.rubies/ruby-2.3.4/bin/ruby"
[2] pry(main)> Gem.ruby
=> "/Users/remy/.rubies/ruby-2.3.4/bin/ruby"
```
> Then again, I seem to have mischecked. While they both seem to return the running Ruby, they return different paths. When installed through Homebrew, for example:

Isn't `/usr/local/opt/ruby/bin/ruby24` a symlink to `/usr/local/Cellar/ruby/2.4.2/bin/ruby24`?

Reply:
On my system, it is a syslink indeed, but I don't know where it gets that when I start `/usr/local/Cellar/ruby/2.4.2/bin/ruby24` directly. I suspect they're both compile time paths somewhere...